src/assets/less/pages/globalping.less

```
@@ -220,31 +333,173 @@
 			}
 
-			&_header {
-				display: none;
+			&_focus-mode {
+				&:after {
+					display: none;
+				}
 
 				@media (min-width: @screen-sm-min) {
+					--focus-mode-offset-y: -398px;
+					transform: translateY(var(--focus-mode-offset-y));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)